bson/src/main/org/bson/types/Decimal128.java

```diff
@@ -18,19 +18,18 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.util.HashSet;
-import java.util.Set;
-
 import static java.math.MathContext.DECIMAL128;
 import static java.util.Arrays.asList;
 import static java.util.Collections.singletonList;
+import java.util.HashSet;
+import java.util.Set;
```

Comment:
Not sure why the import order changed here (vscode auto-adjusted them I guess).

Feel free to put these back if it's an issue